tests/specs/signup-test.js

```diff
@@ -75,12 +81,37 @@ test('successful account.signUp(options)', function (t) {
 
   nock(baseURL)
     .put('/session/account')
-    .reply(200, JSON.stringify(options))
+    .reply(200, JSON.stringify({
+      'links': {
+        'self': 'https://example.com/session/account'
+      },
+      'data': {
+        'id': 'abc4567',
+        'type': 'account',
+        'attributes': options,
+        'relationships': {
+          'profile': {
+            'links': {
+              'related': 'https://example.com/session/account/profile'
+            },
+            'data': {
+              'id': 'abc4567-profile',
+              'type': 'accountProfile'
+            }
+          }
+        }
+      }
+    }))
```

Comment:
I believe I need to `stringify` this JSON because the module expects to `parse` the response from the server. I will move it to a fixtures file though.